### PR TITLE
Add new USPS code for "Arrived at Unit"

### PR DESCRIPTION
**Pull request by ksolkowski**
@fiestacasey @austinkahly @gsnovak

### CODE REVIEW

File: lib/omniship/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Omniship
-  VERSION = '0.3.1'
```
Comment (fiestacasey) [inline lib/omniship/version.rb:2]: need to bundle install

File: lib/omniship/usps/track/package.rb [hunk 2]
```diff
@@ -37,9 +37,11 @@ def has_left?
         def has_arrived?
           # http://about.usps.com/publications/pub97/pub97_i.htm
           activity.any? {|activity|
-            # deliverred or ready for pickup at post office, or Notice Left 
-            ["01", # Delivered*
+            # deliverred or ready for pickup at post office, or Notice Left
+            [
+              "01", # Delivered*
               "17", # Picked Up By Agent, Tendered to Returns Agent
+              "07", # Arrived at Unit
```
Comment (fiestacasey) [inline lib/omniship/usps/track/package.rb:44]: my research and the examples I was able to pull do not lead me to believe that this is supposed to mean the customer has it

File: lib/omniship/usps/track/package.rb [hunk 3]
```diff
@@ -37,9 +37,11 @@ def has_left?
         def has_arrived?
           # http://about.usps.com/publications/pub97/pub97_i.htm
           activity.any? {|activity|
-            # deliverred or ready for pickup at post office, or Notice Left 
-            ["01", # Delivered*
+            # deliverred or ready for pickup at post office, or Notice Left
```
Comment (fiestacasey) [inline lib/omniship/usps/track/package.rb:40]: deliverred only has 1 r in it

**Review by gsnovak — APPROVED** [review]

File: lib/omniship/usps/track/package.rb (hunk 2 above)
Comment (austinkahly) [reply lib/omniship/usps/track/package.rb:44]: ^^^

**Review by gsnovak — CHANGES_REQUESTED** [review]
what casey said

**Pull request closed without merging**